code/lib/builder-vite/src/list-stories.ts

```diff
@@ -11,14 +12,11 @@ export async function listStories(options: Options) {
         configDir: options.configDir,
         workingDir: options.configDir,
       }).map(({ directory, files }) => {
-        const pattern = posixPath.join(directory, files);
+        const pattern = slash(path.join(directory, files));
 
-        return glob(
-          posixPath.isAbsolute(pattern) ? pattern : posixPath.join(options.configDir, pattern),
-          {
-            follow: true,
-          }
-        );
+        return glob(path.isAbsolute(pattern) ? pattern : path.join(options.configDir, pattern), {
```

Comment:
good question, I'll add it and let the tests run. 

BTW Isn't there anything testing that we only pass forward slashes to glob? (the reason for the change which broke windows)

Reply:
Our CI does not do a good job testing Windows, unfortunately.  I think it's something we should investigate.

By the way, if you didn't already know, when you're testing this out locally, a good way is to run

```shell
yarn task --task sandbox 
```

This will generate a project using your code, which you can then navigate into and run `yarn storybook` or `yarn build-storybook` to make sure that the project starts and runs correctly.

Reply:
I did change the code, looks tidier now IMHO and does address the issue you mentioned